main.go

```diff
@@ -244,31 +249,38 @@ func createMessengerInProc(pk *ecdsa.PrivateKey, dbPath string, logger *zap.Logg
 		filterMailTypesHandler(signalsForwarder.in),
 	)
 
-	nodeConfig, err := generateStatusNodeConfig(*dataDir, *fleet, *listenAddr, *configFile)
-	if err != nil {
-		exitErr(errors.Wrap(err, "failed to generate node config"))
-	}
+	var whisper whispertypes.Whisper
+	if *useNimbus {
+		whisper = newNimbusWhisperWrapper()
```

Comment:
any error handling needed here?

Reply:
None needed because this is only a wrapper to the node.